src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/MediaCollection.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React from 'react';
+import {observable} from 'mobx';
+import {observer} from 'mobx-react';
+import {Divider} from 'sulu-admin-bundle/components';
+import {Datagrid, DatagridStore} from 'sulu-admin-bundle/containers';
+import CollectionInfoStore from './stores/CollectionInfoStore';
+import BreadcrumbBuilder from './BreadcrumbBuilder';
+
+type Props = {
+    page: observable,
+    locale: observable,
+    mediaViews: Array<string>,
+    mediaStore: DatagridStore,
+    collectionStore: DatagridStore,
+    collectionInfoStore: CollectionInfoStore,
```

Comment:
Didn't we say that we call that `collectionInfoStore` `collectionStore` instead? And rename the upper ones therefore to `mediaDatagridStore` and `collectionDatagridStore`.